tests/src/test/java/alluxio/underfs/orangefs/OrangeFSUnderStorageCluster.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the “License”). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.underfs.orangefs;
+
+import alluxio.Configuration;
+import alluxio.Constants;
+import alluxio.underfs.UnderFileSystemCluster;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.IOException;
+
+public class OrangeFSUnderStorageCluster extends UnderFileSystemCluster {
+
+  public OrangeFSUnderStorageCluster(String baseDir, Configuration configuration) {
+    super(baseDir, configuration);
+    checkOrangeFSConfigured(configuration);
+  }
+
+  private void checkOrangeFSConfigured(Configuration conf) {
+    if (conf == null) {
+      throw new NullPointerException("Null Alluxio Configuration provided");
```

Comment:
I think `IllegalArgumentException` is more consistent with the rest of the code base.